src/mlpack/methods/lsh/lsh_search.hpp

```diff
@@ -320,6 +345,25 @@ class LSHSearch
                       const size_t neighbor,
                       const double distance) const;
 
+  /**
+   * This function implements the core idea behind Multiprobe LSH. It is called
+   * by ReturnIndicesFromTables when T > 0. Given a query's code and its
+   * projection location, GetAdditionalProbingBins will calculate the T most
+   * likely alternative bin codes (other than queryCode) where a query's
+   * neighbors might be found in.
+   *
+   * @param queryCode vector containing the numProj-dimensional query code.
+   * @param queryCodeNotFloored vector containing the projection location of the
+   * query.
+   * @param T number of additional probing bins.
+   * @param additionalProbingBins matrix. Each column will hold one additional
+   * bin.
+  */
+  void GetAdditionalProbingBins(const arma::vec& queryCode,
+                            const arma::vec& queryCodeNotFloored,
+                            const size_t T,
+                            arma::mat& additionalProbingBins) const;
```

Comment:
The spacing seems off here, the second/third/fourth arguments should align with the first.